distrobox: add & refresh page (#9143)
tldr-pages/tldr: pages/linux/distrobox-rm.md

# distrobox-rm

> Remove a distrobox container.

> > Subcommand of `distrobox`. More about distrobox: `tldr distrobox`.

Reviewer: ```suggestion
> Subcommand of `distrobox`. See also: `tldr distrobox`.
```

> More information: <https://github.com/89luca89/distrobox/blob/main/docs/usage/distrobox-rm.md>.

> - Remove a distrobox (remember to stop container before removing it):

Reviewer: ```suggestion
- Remove a distrobox (Tip: Stop the container before removing it):
```
How about this?
